packages/react-components/src/video-player/utils/dateRangeUtils.tsx

```diff
@@ -0,0 +1,46 @@
+import { DateRangePickerProps } from '@awsui/components-react';
+
+export const getStartAndEndTimeFromRange = (
+  newDateRange: DateRangePickerProps.AbsoluteValue | DateRangePickerProps.RelativeValue,
+  currentDateTimeForRelativeValue?: Date
+) => {
+  let startAndEndTime = undefined;
+  if ('relative' === newDateRange.type) {
+    let offset = 0;
+    switch (newDateRange.unit) {
+      case 'second':
+        offset = newDateRange.amount;
+        break;
+      case 'minute':
+        offset = newDateRange.amount * 60;
```

Comment:
can we utilize constants like we do in the other packages, i.e. `MINUTE_IN_MS`, `DAY_IN_MS`, etc.